src/ErrorBoundary/ErrorBoundary.test.js

```diff
@@ -0,0 +1,5 @@
+// import React from 'react';
```

Comment:
We'll do this next! Maybe that try/catch error example we were using to test the render with manually?